pom.xml

```diff
@@ -146,6 +146,13 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
+		
+		<!--  REMOTE -->
+		<dependency>
+		    <groupId>org.eclipse.paho</groupId>
+		    <artifactId>org.eclipse.paho.client.mqttv3</artifactId>
+		    <version>1.1.0</version>
```

Comment:
Could you use at least version 1.2.1 of this MQTT client library? There's a security flaw in this version, see Snyk analysis of this commit.

Reply:
Thanks. I changed the version now. I resolve this.